### DISCUSSION TITLE
API for Client

### DISCUSSION THREAD

**Discussion by paulobreim** [DinoLeung/TeleDart#240] [General]
Hi @DinoLeung 

Do you have API to use as client  (TDLIB) ?

For example:
I would like to make a program that was a telegram user and was inside a group analyzing the messages.
Since it's not a group I created, I can't use it as a BOT

tks 
Paulo

**Comment by DinoLeung** [comment]
there's no plans to implement a telegram client at this point in time, i barely have time to maintain teledart
